azure-pipelines.yml

```diff
@@ -43,10 +43,29 @@ stages:
       buildOptions: 'USERNAME=admin SONIC_BUILD_JOBS=$(nproc) ${{ variables.VERSION_CONTROL_OPTIONS }}'
       jobGroups:
```

Comment:
I see these lines repeated in below stage. Can we reuse?

Reply:
I can check if we can reuse it, it depends on if the variables can be used another variable in compiling time.